### DISCUSSION TITLE
tar on windows hangs (no output - eventually times-out)

### DISCUSSION THREAD

**Discussion by mellery451** [community/community#25033] [Actions]
<p>I have a fairly straightforward tar command that I run on other platforms (mac/linux) without issue, but it hangs with no output on windows:</p>
<p><a href="https://github.com/mellery451/rippled/runs/413436671?check_suite_focus=true#step:6:4524" rel="nofollow noopener">https://github.com/mellery451/rippled/runs/413436671?check_suite_focus=true#step:6:4524</a></p>
<p>I’m not sure how to debug further, but I have tried adding ACTIONS_STEP_DEBUG/ACTIONS_RUNNER_DEBUG which don’t seem to tell me much. Is there some other technique to figure out what is going on with the windows machine at a particular step?</p>

**Comment by thboop** [comment, marked as answer]
<p>Hey <a class="mention" href="/mellery451">@mellery451</a>,</p>
<p>This is a known issue with a recent update with the virtual environments. A fix is underway, here’s the issue for reference:<br>
<a href="https://github.com/actions/virtual-environments/issues/263" rel="nofollow noopener">https://github.com/actions/virtual-environments/issues/263</a></p>
<p>If you are looking for a quick workaround, here’s what we did in the Actions Runner:</p>
<p><a href="https://github.com/actions/runner/pull/285/files" rel="nofollow noopener">https://github.com/actions/runner/pull/285/files</a></p>
